presto-tests/src/main/java/com/facebook/presto/tests/datatype/DataType.java

```diff
@@ -58,6 +60,22 @@
         return dataType(insertType, prestoResultType, DataType::quote, Function.identity());
     }
 
+    public static DataType<String> charDataType(int length)
+    {
+        return charDataType(length, "");
+    }
+
+    public static DataType<String> charDataType(int length, String properties)
```

Comment:
It's sometimes confusing when reading the tests what are arguments of `charDataType` or other dataType static constructor functions. E.g. once `charDataType` takes `length` as argument. Other time it takes `insertType`. Sometimes it takes `property`.

Maybe it would be better to have `DataType` builder instead with explicit functions:
`setInsertType`, `setLength` etc...